content/en/tracing/profiler/profiler_troubleshooting.md

```diff
@@ -117,9 +124,69 @@ Override templates let you specify profiling properties to override. However, th
     java -javaagent:/path/to/dd-java-agent.jar -Ddd.profiling.enabled=true -Ddd.logs.injection=true -Ddd.trace.sample.rate=1 -Ddd.profiling.jfr-template-override-file=</path/to/override.jfp> -jar path/to/your/app.jar
     {{< /code-block >}}
 
-## Further Reading
+{{< /programming-lang >}}
+{{< programming-lang lang="python" >}}
 
-{{< partial name="whats-next/whats-next.html" >}}
+## Missing profiles in the profile search page
+
+If you've configured the profiler and don't see profiles in the profile search page, turn on [debug mode][1] and [open a support ticket][2] with debug files and the following information:
+
+- Operating system type and version (for example, Linux Ubuntu 20.04)
+- Runtime type, version, and vendor (for example, Python 3.9.5)
+
+[1]: /tracing/troubleshooting/#tracer-debug-logs
+[2]: /help/
+
+{{< /programming-lang >}}
+{{< programming-lang lang="go" >}}
+
+## Missing profiles in the profile search page
+
+If you've configured the profiler and don't see profiles in the profile search page, turn on [debug mode][1] and [open a support ticket][2] with debug files and the following information:
+
+- Operating system type and version (for example, Linux Ubuntu 20.04)
+- Runtime type, version, and vendor (for example, Go 1.16.5)
+
+[1]: /tracing/troubleshooting/#tracer-debug-logs
+[2]: /help/
+
+{{< /programming-lang >}}
+{{< programming-lang lang="ruby" >}}
+
+## Missing profiles in the profile search page
+
+If you've configured the profiler and don't see profiles in the profile search page, turn on [debug mode][1] and [open a support ticket][2] with debug files and the following information:
+
+- Operating system type and version (for example, Linux Ubuntu 20.04)
+- Runtime type, version, and vendor (for example, Ruby 2.7.3)
+
+## Application triggers "stack level too deep (SystemStackError)" errors
+
+The profiler instruments the Ruby VM to track thread creation.
+This instrumentation is compatible with most other Ruby gems that also instrument thread creation, with a few exceptions.
+
+If you're using any of the below gems:
+
+* `rollbar`: Ensure you're using version 3.1.2 or newer.
+* `logging`: Disable `logging`'s thread context inheritance by setting the `LOGGING_INHERIT_CONTEXT` environment
+  variable to `false`.
+
+If you're still experiencing `SystemStackError` errors after following the above instructions,
+[open a support ticket][2] taking care to include the full backtrace leading to the error.
+
+## Missing profiles for Resque jobs
```

Comment:
You can ignore the headings flag below. It's incorrectly flagging as Resque is formal.